ibis/backends/tests/test_window.py

```diff
@@ -642,8 +618,8 @@ def test_simple_ungrouped_unbound_following_window(
 
 @pytest.mark.notimpl(
     ["flink"],
-    raises=com.UnsupportedOperationError,
-    reason="OVER RANGE FOLLOWING windows are not supported in Flink yet",
+    raises=Py4JJavaError,
+    reason="flink doesn't allow order by NULL without casting the null to a specific type",
```

Comment:
Could we order by using a different value?

Reply:
Perhaps, but I'll leave for someone else to do in a follow-up.